README.rst

```diff
@@ -0,0 +1,83 @@
+=============
+Two Qubit Interactions
+=============
+
+This exercise contains three small programs which illustrate two qubit
+interactions in QUBO models. Understanding these small problems is vital for
+writing QUBOs for much larger problems.
+
+Exercise 1 
+----------
+
+Run the command
+
+.. code-block:: bash
+
+  python two_same.py
+
+Read through the code and take a look at the
+structure of the program. Notice the basic parts:
+
+- Obtain a sampler/solver
+- Define the Q matrix
+- Run the problem, using the sampler/solver
+- Print the results
+
+In this exercise, we want to penalize the two qubit solutions in which the 
+qubits have different values; we want to favor the two qubit solutions in 
+which the qubits have the same values.
+
+In the results, you should notice that the solutions (0 0) and (1 1) have 
+lower energy than the solutions (0 1) and (1 0).
+
+Exercise 2 
+----------
+
+Run the command
+
+.. code-block:: bash
+
+  python two_different.py
+
+In this exercise, we want to penalize the two qubit solutions in which the 
+qubits have the same values; we want to favor the two qubit solutions in 
+which the qubits have different values.
+
+In the results, you should notice that the solutions (0 1) and (1 0) have 
+lower energy than the solutions (0 0) and (1 1).
+
+Exercise 3 
+----------
+
+Run the command
+
+.. code-block:: bash
+
+  python two_implies.py
+
+The implies function in logic has the following truth table:
+
+=== === ==========
+X   Y   result
+--- --- ----------
+F   F   T
+F   T   T 
+T   F   F
+T   T   T
+=== === ==========
+
+
+In QUBO variables, we can write the problem as
+
+=== === ==========
+q_1 q_2 result
+--- --- ----------
+0   0   0
+0   1   0
+1   0   1
+1   1   0
```

Comment:
If the QUBO is suppose to represent the implies function truth table, then one of your truth tables is incorrect. (i.e. I expected the `F -> 0` and `T -> 1`.